src/components/search-form/search-form.js

```diff
@@ -75,11 +79,11 @@ class SearchForm extends Component {
    * for the current searchType
    */
   getFiltersComponent = (searchType) => {
-    if (searchType === 'titles') {
+    if (searchType === searchTypes.TITLES) {
       return TitleSearchFilters;
-    } else if (searchType === 'packages') {
+    } else if (searchType === searchTypes.PACKAGES) {
       return PackageSearchFilters;
-    } else if (searchType === 'providers') {
+    } else if (searchType === searchTypes.PROVIDERS) {
       return ProviderSearchFilters;
```

Comment:
Maybe it would be cleaner to make it like this:
```javascript
const filterComponents = {
  [searchTypes.TITLES]: TitleSearchFilters,
  [searchTypes.PACKAGES]: PackageSearchFilters,
  [searchTypes.PROVIDERS]: ProviderSearchFilters,
}
```

and then we could write 
```js
const filterComponent = filterComponents[searchType] || null;
```

And I think that probably `null` is not needed there which means it can be even more clean:
```js
const filterComponent = filterComponents[searchType];
```